adobe_vipm/flows/fulfillment/reseller_transfer.py

```diff
@@ -181,3 +182,35 @@ def __call__(self, mpt_client, context, next_step):
             context.agreement_id,
             externalIds={"vendor": context.adobe_customer_id},
         )
+
+
+class UpdateAutorenewalSubscriptions(Step):
+    """Updates the auto renewal status of the subscriptions."""
+
+    def __call__(self, mpt_client, context, next_step):
+        """Updates the auto renewal status of the subscriptions."""
+        adobe_client = get_adobe_client()
+        subscriptions = adobe_client.get_subscriptions(
+            context.authorization_id,
+            context.customer_id,
+        ).get("items", [])
+
+        for subscription in subscriptions:
+            if subscription.get("autoRenewal", {}).get("enabled", True):
+                continue
+
+            subscription_id = subscription["subscriptionId"]
+            try:
+                adobe_client.update_subscription(
+                    context.authorization_id,
+                    context.customer_id,
+                    subscription_id,
+                    auto_renewal=True,
+                )
+            except AdobeAPIError:
+                logger.exception(
```

Comment:
Should this be a warning or info instead of an exception?  We have two kind of subscriptions that are not auto-renewal-> one-time items and non-renewal subscriptions. One-time are easy to discard, but not the non-renewal subscriptions